app/main/helpers/helpers.py

```diff
@@ -0,0 +1,20 @@
+from flask import abort
+from dmutils.apiclient import APIError
+
+from ... import data_api_client
+
+
+def call_data_api_client(method_name, **kwargs):
```

Comment:
I don't think this is a good approach, it means we have to change how we call the API client everywhere. I'd prefer something like @allait suggested where we do this with an error handler. see: http://flask.pocoo.org/docs/0.10/patterns/apierrors/